knife/spec/unit/knife/bootstrap_spec.rb

```diff
@@ -697,6 +696,33 @@ def configure_gem_files
         expect { knife.validate_protocol! }.to raise_error SystemExit
       end
     end
+
+    context "when additional Train transports are present" do
+      before do
+        Gem::Specification.new do |spec|
+          spec.name = "train-rfc2549"
+          spec.version = "0.0.1"
+          spec.summary = "Train transport to use IPoAC"
+        end.activate
+
+        Gem.refresh
+      end
+
+      context "and their usage is supported" do
+        let(:connection_protocol) { "rfc2549" }
+        it "accepts the transport as protocol" do
+          expect(knife.validate_protocol!).to eq true
+        end
+      end
+
+      context "and invalid proctocols are still revused" do
```

Comment:
```suggestion
      context "and invalid proctocols are still refused" do
```

Reply:
@tecracer-theinen did you address the spelling here? Looks like "revused" is still there instead of "refused"